python/pylibcudf/pylibcudf/scalar.pyx

```diff
@@ -25,6 +34,13 @@ from .types cimport DataType
 
 from functools import singledispatch
 
+try:
+    import numpy as np
+    np_error = None
+except (ImportError, ModuleNotFoundError) as err:
```

Comment:
FWIW ModuleNotFoundError inherits from ImportError so unless you're going to handle the two of them differently, this clause is currently redundant.